src/plugins/data/common/index_patterns/types.ts

```diff
@@ -185,6 +185,8 @@ export interface FieldSpec {
   readFromDocValues?: boolean;
   subType?: IFieldSubType;
   indexed?: boolean;
+  autoCompleteValues?: string[];
+  topQueryValues?: string[];
```

Comment:
What if instead of two new fields we make one new field that is a object that has these two values set on them. The reason why is because I feel like these values should be in sync with each other and updated together